bindings/jupyter/Makefile

```diff
@@ -9,15 +9,22 @@ install:
 clean:
 	rm -rf dist
 
-build:
+build-js:
+	cd js && yarn build
+
+build-py:
 	rm -rf dist
 	python setup.py sdist bdist_wheel
 
+build:
+	cd js && yarn build
+	rm -rf dist
+	python setup.py sdist bdist_wheel
```

Comment:
Can we just make use of `build-js` and `build-py` here?

Reply:
Tweaked the make file to call rules within rules.